eks/modules/control_plane/main.tf

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
Is this enter needed? Have you run terraform fmt over your code from PR?

Reply:
Yes I have, it was never catched by `fmt`, but I will remove them.